tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -69,8 +68,14 @@ def test_node_maintenance(self, node_type, pvc_factory, pod_factory):
 
         """
         # Get 1 node
-        typed_nodes = get_typed_nodes(node_type=node_type, num_of_nodes=1)
+        typed_nodes = get_typed_nodes(node_type=node_type, num_of_nodes=2)
```

Comment:
the comment above should be changed as well?

Reply:
correct, will fix